granjaYcosas.wlk

```diff
@@ -0,0 +1,128 @@
+import hector.*
+import cultivos.*
+import wollok.game.*
+
+object granja {
+  
+  const property cultivos =  #{}
+  const property construcciones = #{mercado1, mercado2} 
+  const property cosasJardineria = #{}
+
+  //sembrar:
+  method agregarACultivo(position, planta) {
+    planta.position(position)
+    game.addVisual( planta )
+    cultivos.add(planta)
+  }
+
+  method esEspacioVacio(position) { 
+    return not self.hayPlantasAqui(position) and not self.hayMercadosAqui(position) and not self.hayCosasDeJardineriaAqui(position) 
+	}
```

Comment:
Bien TIP: muchas negaciones! jaja hay una ley lógica de equivalencia, que se puede poner no es verdad que: esto o esto otro o esto último:
not (self.hayPlantasAqui(position) or self.hayMercadosAqui(position) or  self.hayCosasDeJardineriaAqui(position) )